src/libawkward/array/RegularArray.cpp

```diff
@@ -238,7 +238,7 @@ namespace awkward {
   }
 
   const std::shared_ptr<Content> RegularArray::flatten(int64_t axis) const {
-    throw std::runtime_error("FIXME: not implemented");
+    return content_;
```

Comment:
This is correct, though there's a tiny corner-case: if `len(content) % size != 0`, then the `RecordArray` is a little shorter than its content. For consistency, this should be

```c++
content_.get()->getitem_range_nowrap(0, length()*size_)
```

which will cut off the excess that might be lurking in the `content_`.

See the [definition of RecordArray](https://github.com/scikit-hep/awkward-1.0/wiki/RegularArray.md).